capella.go

```diff
@@ -0,0 +1,158 @@
+package relay_grpc
+
+import (
+	"github.com/attestantio/go-builder-client/api/capella"
+	v1 "github.com/attestantio/go-builder-client/api/v1"
+	"github.com/attestantio/go-eth2-client/spec/bellatrix"
+	consensus "github.com/attestantio/go-eth2-client/spec/capella"
+	"github.com/attestantio/go-eth2-client/spec/phase0"
+	"github.com/holiman/uint256"
+)
+
+func CapellaRequestToProtoRequest(block *capella.SubmitBlockRequest) *SubmitBlockRequest {
```

Comment:
think you should save the "version" in as a member of `SubmitBlockRequest` struct in these functions

Reply:
Agreed, will make this change